### ISSUE TITLE
Accessibility: Screen Reader Fails to Announce Notification Status  "4.1.3 Status Messages" Violation

### ISSUE THREAD

**Issue by anitakumari28** [microsoft/vscode#249426]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.100.0
- OS Version: macOS Sequoia 15.4.1
- VoiceOver: Built-in macOS Screen reader

Steps to Reproduce:

1. On macOS, press Command + F5 to activate the built-in VoiceOver screen reader (or activate any screen reader of choice).
2. Click on the notification icon in VS Code to read the latest notification message.

When a notification is opened, the screen reader reads the message content directly without first announcing the type or status of the notification (e.g., "Info", "Warning", or "Error").

According to Requirement No. [4.1.3 Status Messages](https://www.w3.org/WAI/WCAG22/Understanding/status-messages), screen readers should first announce the type/status of a notification before its content. This allows users to understand the context or severity of the message prior to hearing the details.

<img width="1727" alt="Image" src="https://github.com/user-attachments/assets/69dee558-0d76-4068-b041-1f40880673f6" />

This issue affects the overall accessibility of both Visual Studio Code and the WCA4Z VSCode extension, making them less usable for users with visual impairments.
